objectivec/ProtocolBuffers_iOS.xcodeproj/project.pbxproj

```diff
@@ -1090,6 +1281,95 @@
 			};
 			name = Release;
 		};
+		FF1C791D245340E40099AB0D /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
```

Comment:
Please remove the local settings from the targets that aren't explicitly needed.  The project level should be setting most of the warning related flags, we don't want different warnings between the different targets.

Reply:
Removed all unnecessary local settings from here, only ones left should be the needed ones for the framework to build correctly.

Reply:
GCC_PREPROCESSOR_DEFINITIONS repeats `DEBUG`, probably should be $(inherited) and the framework imports one. GCC_C_LANGUAGE_STANDARD should be the same for all targets.  Is CLANG_ENABLE_MODULE really needed?

Reply:
Fixed! I found a few other things I missed in the tvOS and MacOS projfiles as well.
Short answer for CLANG_ENABLE_MODULE, its not needed. TLDR: It's used in frameworks for compiler optimization to make importing cheap but also extensively import all headers in one shot.

Here is some more information on what it does: https://clang.llvm.org/docs/Modules.html#id10

When the project is built with Cocoapods this feature is supported, it's a standard now at this point, I'd also argue that it's required for swift files to be able to import this but I think in most cases the apple/swift-protobuf library is the better option.